Allow Fleet Status to report correct nucleus  version (#757)
aws-greengrass/aws-greengrass-lite: docs/examples/sample.ggLitePython/README.md

### Pre-installed python3 and python3-venv.

- As a python demo it requires the python3 interpreter (if not already
  installed) as a well as a python3 virtual environment for isolated dependency
  management.

> - Alternatively, you may deploy the following recipe deployment before deploying

Reviewer: ```suggestion
- Alternatively, you may deploy the following recipe before deploying
```

>   given python demo.

Reviewer: ```suggestion
  the given python demo.
```

## End result

The logs for following the deployment status can be found by:

```shell
$ journalctl -xeau ggl.core.ggdeploymentd.service
```

> Once successful, use following command to check the log and see the list of

Reviewer: ```suggestion
Once successful, use the following command to check the logs and see the list of
```

> buckets names in your AWS S3 that exist in your aws account.

Reviewer: ```suggestion
bucket names that exist in your AWS account within S3.
```
